### PR TITLE
Revamped guest welcome message

### PR DESCRIPTION
**Pull request by colophonemes**

### CODE REVIEW

File: packages/lesswrong/components/common/Logo.jsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ const styles = theme => ({
   }
 })
 
-const Logo = ({classes}) => <img
+const Logo = ({classes}) => getSetting('logoUrl', null) && <img
```
Comment (jpaddison3) [inline packages/lesswrong/components/common/Logo.jsx]: I like this pattern better tbh, but then you should remove the getSetting && logo in the Header.

File: packages/lesswrong/components/common/GuestWelcomeScreen.jsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+import { Components, registerComponent } from 'meteor/vulcan:core';
+import { withStyles, withTheme } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import Grid from '@material-ui/core/Grid';
+
+
+const styles = theme => ({
+  root: {
+    a: {
+      textDecoration: 'underline'
```
Comment (jpaddison3) [inline packages/lesswrong/components/common/GuestWelcomeScreen.jsx]: I still don't see underlines on the links.

File: packages/lesswrong/components/common/GuestWelcomeScreen.jsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  },
+  body: {
+    fontSize: 18
+  },
+  logoWrapper: {
+    textAlign: 'center',
+    marginBottom: theme.spacing.unit
+  },
+  logo: {
+    display: 'inline-block'
```
Comment (jpaddison3) [inline packages/lesswrong/components/common/GuestWelcomeScreen.jsx]: What does this do?

Comment (colophonemes) [reply packages/lesswrong/components/common/GuestWelcomeScreen.jsx]: As discussed

**Pull request merged**